vue-client/src/resources/json/combinedTemplates.json

```diff
@@ -964,6 +964,200 @@
+                {
+                  "@type": "Place",
+                  "label": [
+                    ""
+                  ]
+                }
+              ]
+            }
+          ],
+          "carrierType": [{"@id": "https://id.kb.se/term/rda/Volume"}],
+          "baseMaterial": [{"@id": "https://id.kb.se/term/rda/material/Paper"}],
+          "extent": [
+            {
+              "@type": "Extent",
+              "label": ""
+            }
+          ],
+          "physicalDetailsNote": "illustrationer",
+          "productionMethod": [
+            {
+              "@type": "ProductionMethod",
+              "label": [
+                "handskrift"
+              ]
+            }
+          ],
+          "hasDimensions": {
+            "@type": "Dimensions",
+            "label": ""
+          },
+          "hasNote": [
+            {
+              "@type": "Note",
+              "label": [
+                "Ingår i KB:s handskriftssamling"
+              ]
+            },
+            {
+              "@type": "Note",
+              "label": ""
+            }
+          ],
+          "instanceOf": {
+            "@type": "Text",
+            "language": [
+              {
+                "@id": "https://id.kb.se/language/swe"
+              }
+            ],
+            "genreForm": [
+              {
+                "@id": "https://id.kb.se/term/saogf/Handskrifter"
+              }
+            ],
+            "contribution": [
+              {
+                "@type": "PrimaryContribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/author"
+                  }
+                ]
+              },
+              {
+                "@type": "Contribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/author"
+                  }
+                ]
+              },
+              {
+                "@type": "Contribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/unspecifiedContributor"
+                  }
+                ]
+              }
+            ],
+            "contentType": [
+              {
+                "@id": "https://id.kb.se/term/rda/Text"
+              }
+            ],
+            "hasNote": [
+              {
+                "@type": "marc:LanguageNote",
+                "label": ""
+              }
+            ],
+            "subject": [
+              {
+                "@type": "Person",
+                "familyName": "",
+                "givenName": "",
+                "lifeSpan": ""
+              },
+              {
+                "@type": "ComplexSubject",
+                "inScheme": {
+                  "@id": "https://id.kb.se/term/sao"
+                },
+                "termComponentList": [
```

Comment:
Untimely perhaps, but given that we're working to get rid of at least complex subjects with geographic terms, should we "discourage" them now but not putting them in templates?

Reply:
I know! If we can land that before or very close to the next release it we should ofc remove it, otherwise it is a big time-saver (unfortunately), my reasoning now is to maybe keep it until we can be more sure.

Reply:
I agree. Would you like to create a  issue/task in the Geo story for removing complex-geo-subjects for _all_ templates?

Reply:
Subtask created ✅